.github/workflows/release_docker.yml

```diff
@@ -2,8 +2,6 @@ name: Publish Docker Image
 
 on:
   push:
-    branches:
-      - "main"
```

Comment:
We don't want a `latest` docker image that tracks main (for testing, etc)?

Reply:
Ah yeah, good point. Two questions about it.

Are there changes that are not user facing that should trigger a new Docker image? With the current setup, we will create a new image for every `feat:`, `fix:`, and breaking change. Are there cases where we would want them for `chore:`, `build:`, etc?

Second, does `ghcr` have a mechanism for listing images by `git` tag? They don't list our existing tags ((https://github.com/Lilypad-Tech/lilypad/pkgs/container/resource-provider/versions), but maybe it's because of the way we have been formatting them. The concern here is whether or not resource providers can find the latest released version vs the latest published version.

Reply:
> Are there changes that are not user facing that should trigger a new Docker image? With the current setup, we will create a new image for every `feat:`, `fix:`, and breaking change. Are there cases where we would want them for `chore:`, `build:`, etc?

Maybe not? I think I don't know the answer yet.

> Second, does `ghcr` have a mechanism for listing images by `git` tag? They don't list our existing tags ((https://github.com/Lilypad-Tech/lilypad/pkgs/container/resource-provider/versions), but maybe it's because of the way we have been formatting them. The concern here is whether or not resource providers can find the latest released version vs the latest published version.

It will publish on git tags - I think we just haven't done one since the docker build step was added.

Reply:
Alright yeah, if the tagged versions will be distinct on `ghcr`, let's publish images on push to `main` and tags. Will drop the change.

Can imagine a scenario where we are updating the Dockerfile, it's a `build:` task not a fix/feature, and we want the team to have easy access to an image.